sympy/physics/continuum_mechanics/truss.py

```diff
@@ -160,36 +156,45 @@ def add_node(self, label, x, y):
 
         >>> from sympy.physics.continuum_mechanics.truss import Truss
         >>> t = Truss()
-        >>> t.add_node('A', 0, 0)
+        >>> t.add_node(('A', 0, 0))
         >>> t.nodes
         [('A', 0, 0)]
-        >>> t.add_node('B', 3, 0)
+        >>> t.add_node(('B', 3, 0), ('C', 4, 1))
         >>> t.nodes
-        [('A', 0, 0), ('B', 3, 0)]
+        [('A', 0, 0), ('B', 3, 0), ('C', 4, 1)]
         """
-        x = sympify(x)
-        y = sympify(y)
 
-        if label in self._node_labels:
-            raise ValueError("Node needs to have a unique label")
+        for i in args:
+            label = i[0]
+            x = i[1]
+            x = sympify(x)
+            y=i[2]
+            y = sympify(y)
+            if label in self._node_labels:
```

Comment:
Instead of `_node_labels`, try using a dictionary like `_node_coordinates` for the same because of the lookup time complexity in a dictionary